doc/getting_started.rst

```diff
@@ -235,7 +235,41 @@ Now you can use **Lua Fun**:
     2 c
     one
 
-Please note that functions support multireturn.
+Please note that functions support multireturn. It means that some of them may
+return more than one iterator, for example functions ``partition`` and ``span``.
+
+One call ``:unwrap()`` on generators to pass them into the ``for <..> in <...>``
+loop. Example:
```

Comment:
I would prefer to call luafun iterators as 'iterators', not generators. At least to don't confuse with a subset of iterators that are generators. And to don't confuse with just `gen` function.

BTW, 'one call :unwrap()' seems lacks a verb if I don't miss anything.

Reply:
Fixed.

```asciidoc
--- a/doc/getting_started.rst
+++ b/doc/getting_started.rst
@@ -238,8 +238,8 @@ Now you can use **Lua Fun**:
 Please note that functions support multireturn. It means that some of them may
 return more than one iterator, for example functions ``partition`` and ``span``.
 
-One call ``:unwrap()`` on generators to pass them into the ``for <..> in <...>``
-loop. Example:
+One can call ``:unwrap()`` on iterators to pass them into the ``for <..> in
+<...>`` loop. Example:
 
 .. code-block:: bash
 
```

Reply:
![Screenshot from 2021-11-29 21-21-11](https://user-images.githubusercontent.com/1151557/143921990-d16825fc-136d-4412-9bb3-7c5bdaa02d2a.png)